core/invoker/src/main/scala/whisk/core/containerpool/ContainerProxy.scala

```diff
@@ -94,7 +94,7 @@ case object ActivationCompleted
  */
 class ContainerProxy(
     factory: (TransactionId, String, ImageName, Boolean, ByteSize) => Future[Container],
-    sendActiveAck: (TransactionId, WhiskActivation) => Future[Any],
+    sendActiveAck: (TransactionId, WhiskActivation) => Unit,
```

Comment:
Why don't we preserve the interface here? I know it's unused but why change it if not needed?